.env.dist

```
@@ -1,6 +1,7 @@
 # test, dev, prod
 APP_ENV=test
-APP_PORT=8181
+APP_PORT=8282
+HTTP_APP_PORT=8181
```

Maybe it will be better to add ability to specify listening address, for example
```suggestion
HTTP_ADDRESS=localhost:8181
```